go/libraries/doltcore/table/untyped/csv/reader.go

```diff
@@ -39,10 +43,25 @@ var ReadBufSize = 256 * 1024
 type CSVReader struct {
 	closer io.Closer
 	bRd    *bufio.Reader
-	info   *CSVFileInfo
 	sch    schema.Schema
 	isDone bool
 	nbf    *types.NomsBinFormat
+
+	// CSV parsing is based on the standard Golang csv parser in encoding/csv/reader.go
+	// This parser has been adapted to differentiate between quoted and unquoted
+	// empty strings, and to use multi-rune delimiters. This adaptation removes the
+	// comment feature and the lazyQuotes option
+	delim   []byte
+	numLine int
+	// rawBuffer is a line buffer only used by the readLine method.
+	rawBuffer []byte
```

Comment:
why is this a member of the struct and not just a local inside readLine?  I'd be surprised if reusing this buffer was a big performance benefit... but maybe on csvs that have really wide rows.